types/experiments.ts

```diff
@@ -0,0 +1,81 @@
+/** All fields are publically accessible via the API and available to clients */
+interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  name: string;
+  /** Short public description of the experiment */
+  public_description: string;
+
+  /** Current state of the experiment. */
+  status: "Draft" | "Review" | "Accepted" | "Rejected" | "Live" | "Complete";
```

Comment:
What's the difference between "Accepted" and "Live"?

My thought was that during experiment creation the status is "Draft".
As soon as you submit it for approval (by Remote Settings moderators) it gets put into "Review" state.
Remote Settings moderators Accept or Reject the experiment.
But I was assuming that once an experiment is approved, it'd immediately be live.
And then Complete makes sense for the end of the experiment time period.

If an experiment is rejected by a moderator, can the submitter edit the experiment and resubmit for re-review, or would they need to enter the information again as a new experiment?